### DISCUSSION TITLE
How to add custom hairstyles

### DISCUSSION THREAD

**Discussion by fk20268** [dapilab/react-nice-avatar#32] [Q&A] [answered]
Hi, sorry to bother you again but would you have any advice on how I would customise this so I could add images of different hairstyles or shirt styles to the component?

**Comment by wwayne** [comment, marked as answer]
You can add your component into the folder `src/hair` and edit the file `src/hair/index.tsx` to accept a new hairstyle, after that you should need to update the typescript type to accept the newly added component.

In the beginning, I generate those components by exporting SVG from the Figma files and did some modifications manually
